.github/ISSUE_TEMPLATE/bug_template.md

```diff
@@ -6,12 +6,10 @@ labels: web-ui
 
```

Comment:
Could we add the `issue: bug` label as well? 🙏 https://github.com/sensu/web/labels/issue%3A%20bug